pandasai/smart_datalake/__init__.py

```diff
@@ -255,7 +255,12 @@ def _get_cache_key(self) -> str:
 
         return cache_key
 
-    def chat(self, query: str, output_type: Optional[str] = None):
+    def chat(
+        self,
+        query: str,
+        output_type: Optional[str] = None,
+        start_conversation: Optional[str] = None,
```

Comment:
Let's remove this

Reply:
@gventuri  Currently the get_conversation method is called with default which has limit=1 means returning the last message only. We need to then pass the memory size to SmartLake constructor to do so.